scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -289,7 +289,7 @@ def setup_layout(self):
         runs_icon = icons.get_icon("mdi.play-circle-outline")
```

Comment:
Much like with the settings icon can we keep this inline with the other parts of workbench Icons.
```suggestion
        runs_icon = icons.get_icon("mdi.play")
```

Reply:
Good suggestion